File: src/main/java/in/dragonbra/javasteam/steam/steamclient/callbackmgr/ICallbackMgrInternals.java
package in.dragonbra.javasteam.steam.steamclient.callbackmgr;

/**
 * @author lngtr
 * @since 2018-02-23
 */
public interface ICallbackMgrInternals {
    void register(CallbackBase callback);

    void unregister(CallbackBase callback);
}

File: src/main/java/in/dragonbra/javasteam/steam/steamclient/callbackmgr/ICallbackMgrInternals.kt
package `in`.dragonbra.javasteam.steam.steamclient.callbackmgr

/**
 * @author lngtr
 * @since 2018-02-23
 */
interface ICallbackMgrInternals {
    fun register(callback: CallbackBase)

    fun unregister(callback: CallbackBase)
}
